x-pack/solutions/observability/plugins/inventory/public/components/entity_group_accordion/grouped_entities_grid.tsx

```diff
@@ -100,14 +107,16 @@ export function GroupedEntitiesGrid({ groupValue }: Props) {
 
   return (
     <EntitiesGrid
-      entities={value.entities}
+      entities={value.entities as InventoryEntity[]}
```

Comment:
Do we need this type assertion?

Reply:
Nope, it was a leftover from a previous main merge, removed 👍